dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -154,12 +156,25 @@ public static void main(String[] arguments)
    * @throws RepositoryException when there is a problem in retrieving jar files
    */
   static List<Path> coordinateToJarPaths(String coordinate) throws RepositoryException {
```

Comment:
coordinateToClasspath?

Reply:
That represents why we want the function. Updated.